src/components/UI/Card/ProductCards/ProductsCards.css

```diff
@@ -175,6 +175,15 @@ h2 {
 	.cart-card > .card-footer > .card-footer-view > .btn-addToCart {
 		padding: 0em !important;
```

Comment:
Suggestion: Try not to use important as much as possible. Instead increase specificity.

Reply:
I did actually as you see but didn't get the result the hierarchy is complicated so important does the work